staging/src/k8s.io/metrics/IMPLEMENTATIONS.md

```diff
@@ -12,7 +12,7 @@
 
 ## Custom Metrics API
 
-***NB: None of the below implemenations are officially part of Kubernetes.
+***NB: None of the below implementations are officially part of Kubernetes.
```

Comment:
This has been fixed in https://github.com/kubernetes/kubernetes/pull/59464 - https://github.com/kubernetes/kubernetes/pull/59464/files#diff-1c26f9847072f97a5b83d3a5f2dbc012. Closing.

/close